matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/query/EventEntityQueries.kt

```diff
@@ -32,10 +31,9 @@ internal fun EventEntity.copyToRealmOrIgnore(realm: Realm, insertType: EventInse
             .equalTo(EventEntityFields.ROOM_ID, roomId)
             .findFirst()
     return if (eventEntity == null) {
-        val canBeProcessed = type != EventType.ENCRYPTED || decryptionResultJson != null
```

Comment:
Is it possible to have a case where the type is not  `EventType.ENCRYPTED` and `decryptionResultJson` is set to `null`? If this case exists, I wonder if we should make it processable?

Reply:
Thanks for the explanation. It made me realize I needed to change some logic in `EventInsertLiveObserver`. I have updated it in [60e838a](https://github.com/vector-im/element-android/pull/7851/commits/60e838a82c4d82b26e1141f79cb7028bab50a7c6).